variables.tf

```diff
@@ -28,3 +28,8 @@ variable "k8s_min_master_version" {
   description = "The minimum version of the master"
   default     = "1.13"
 }
+
+variable "machine_type" {
+  description = "The machine type to provision"
+  default     = "n1-standard-1"
+}
```

Comment:
I understand why this has been done in this pr, but it should really be in a pr of its own.

Reply:
I'll open a PR :+1:

Reply:
https://github.com/ONSdigital/eq-terraform-load-generator/pull/6/files